### PR TITLE
Add global parameter mqtt-vhost mapping functions 

### PR DESCRIPTION
**Pull request by awills96**
For this contribution, I followed the current setup of making the templated get function for global parameters that mirrors the vhost parameter setup. On top of that, also implemented the use of global parameter for port to virtual host mapping per specified by the rabbitmq mqtt plugin: https://www.rabbitmq.com/mqtt.html 

Not sure if there are any other global parameters that are somewhat common that should be included.

### CODE REVIEW

File: src/main/java/com/rabbitmq/http/client/Client.java
Comment (michaelklishin) [inline src/main/java/com/rabbitmq/http/client/Client.java:1264]: Can we please rename this to `getMqttPortToVhostMapping`? "MQTT vhost ports" does not make much sense because MQTT does not have a concept of a virtual host.

File: src/main/java/com/rabbitmq/http/client/Client.java
Comment (michaelklishin) [inline src/main/java/com/rabbitmq/http/client/Client.java:1268]: Same as above, `deleteMqttPortToVhostMapping`

File: src/main/java/com/rabbitmq/http/client/Client.java
Comment (michaelklishin) [inline src/main/java/com/rabbitmq/http/client/Client.java:1272]: Same as above, `setMqttPortToVhostMapping`

**Review by michaelklishin — CHANGES_REQUESTED** [review]
Thank you. This looks reasonable to me and even has integration tests.

I'd like to rename a few methods because if we don't do it now, it will be much harder to rename them once this version of the client API ships in a release.

File: src/main/java/com/rabbitmq/http/client/Client.java
Comment (awills96) [reply src/main/java/com/rabbitmq/http/client/Client.java]: Resolved in latest commit.

**Review by michaelklishin — APPROVED** [review]

**Comment by michaelklishin** [comment]
Let's wait for @acogoluegnes' feedback, this LGTM now.

**Review by acogoluegnes — APPROVED** [review]
LGTM.

**Comment by acogoluegnes** [comment]
Thanks!

@awills96 Do we still need #426 or can we close it?

**Comment by awills96** [comment]
@acogoluegnes Good to close!

**Pull request merged**
